test/integration/testLargeResultSet.js

```diff
@@ -322,4 +324,327 @@ describe('SNOW-743920: Large result set with ~35 chunks', function () {
       }
     });
   });
+
+});
+
+describe('variant parsing statistics', function () {
+  let connection;
+  const tableName = 'VARIANT_TABLE2';
+  const selectVariant = `SELECT id, content
+                         FROM ${tableName} limit 3`;
+  const exampleBigJson = `[
+  {
+    "_id": "6501c357397b66ce47719212",
+    "index": 0,
+    "guid": "e7e0e5d8-82b4-47f7-a2ab-68588c93d81e",
+    "isActive": false,
+    "balance": "$2,611.69",
+    "picture": "http://placehold.it/32x32",
```

Comment:
if we need those data (and we can use it) let's add a reference from where they are